db/schema.rb

```diff
@@ -435,6 +436,12 @@
     t.index ["efl_qualification_type", "efl_qualification_id"], name: "index_elp_on_efl_qualification_type_and_id"
   end
 
+  create_table "feature_metrics_dashboards", force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.jsonb "metrics"
+  end
```

Comment:
Haven't pulled this out into its own deploy because it'll only affect support. 👍🏼 / 👎🏼 ?